### PR TITLE
Add support for the Free Mobile API

### PR DESCRIPTION
**Pull request by Mael-Le-Garrec**
Each Free Mobile client has access to a free SMS API. The service is intended to be used for alerts, exactly what we're looking for here.

The pass can be found in "My Options" in your personnal account.

Example usage: 
 `python3.6 Vaccinbot.py --free_mobile_user user --free_mobile_pass pass`

Tested with my own credentials, works fine. Would maybe need to beautify the text but it's not that important.

### CODE REVIEW

File: Vaccinbot.py [hunk 1]
```diff
@@ -32,16 +32,28 @@
 
 
 # Argument parsing
+def get_args():
+    parser = argparse.ArgumentParser(description="A bot that automatically fetches date from ViteMaDose and finds appointments within 24h.")
+    parser.add_argument("--interval", type=int, help="Time in minutes between queries to ViteMaDose. Default: " + str(PING_INTERVAL_MINUTES), default=PING_INTERVAL_MINUTES)
+    parser.add_argument("--slack-token", type=argparse.FileType(), help="Path to file containing a slack token (activates Slack bot feature).")
+    parser.add_argument("--free-mobile-user", type=str, help="User to be used with the Free Mobile SMS API")
+    parser.add_argument("--free-mobile-pass", type=str, help="Password used for the Free Mobile SMS API")
```
Comment (simonecid) [inline Vaccinbot.py]: Can you change this option to take the password via a file the same I do with the ```--slack-token```? This way passwords are not stored in the history of the command line.

**Comment by simonecid** [comment]
Hi, 

Wow, I am a Free mobile customer myself, and I was not aware of that. Thank you for the PR. If you could address the password comment, I would gladly merge this.

Cheers,
Simone

**Comment by Mael-Le-Garrec** [comment]
I added getpass to avoid having the password in clear on the CLI.
I also changed the text to be more SMS friendly.

**Review by simonecid — APPROVED** [review]

**Comment by simonecid** [comment]
Thanks for the contribution!

Cheers,
Simone

**Pull request merged**